components/camel-zipkin/pom.xml

```diff
@@ -45,6 +45,12 @@
             <artifactId>camel-support</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
Please do not add a new dependency just for checking if a string is empty/black.

You can use Camel's own 

org/apache/camel/util/ObjectHelper.isEmpty

Reply:
Change request done.